modules/command/dc/dc-finalize-import-command.js

```diff
@@ -0,0 +1,196 @@
+const Command = require('../command');
+const Models = require('../../../models');
+const bytes = require('utf8-length');
+const Utilities = require('../../Utilities');
+const { sha3_256 } = require('js-sha3');
+const ImportUtilities = require('../../ImportUtilities');
+const Graph = require('../../Graph');
+
+class DcFinalizeImport extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.logger = ctx.logger;
+        this.remoteControl = ctx.remoteControl;
+        this.config = ctx.config;
+        this.notifyError = ctx.notifyError;
+    }
+
+    /**
+     * Executes command and produces one or more events
+     * @param command
+     */
+    async execute(command) {
+        const { afterImportData, error } = command.data;
+        if (error) {
+            await this._processError(error, command.data.handler_id);
+            return Command.empty();
+        }
+        const response = await this._unpackKeysAndSortVertices(afterImportData);
+        response.handler_id = afterImportData.handler_id;
+
+        await this._finalizeImport(response);
```

Comment:
Considering the _finalizeImport command only writes to the database, it might make sense not to separate it from the execute function.